docs/configuring-a-resource.md

```diff
@@ -557,13 +559,66 @@ p.AddResourceConfigurator("aws_autoscaling_group", func(r *config.Resource) {
 })
 ```
 
+### Initializers
+
+Initializers establish ownership of the supplied Managed resources in Crossplane.
+This typically involves the operations that are run before calling any
+ExternalClient methods.
+
+This configuration option will provide that setting custom initializers for per 
```

Comment:
For the readers of this document, it's important to know that whether they need the tool in given section or not right away. So, I'd start with an example case that is easy to understand, define the thing and then go into details of how they can do it. Something like why -> what -> how.